client/db_test.go

```diff
@@ -0,0 +1,28 @@
+package client_test
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/server"
+)
+
+func ExampleDB() {
+	s := server.StartTestServer(nil)
+	defer s.Stop()
+
+	db := client.Open(s.ServingAddr())
+	db.Put("hello", "world")
+	result, _ := db.Get("hello")
+	fmt.Println(result.Rows[0].ValueString())
+
+	db.Inc(100, "world")
```

Comment:
Yeah, the variadic `Put` is a bit weird. Easy enough to adjust both `Inc` and `Put`, and maybe `Scan`, to not be variadic.